pyrobosim/examples/demo.py

```diff
@@ -122,14 +122,16 @@ def create_world(multirobot=False):
             grasp_generator=GraspGenerator(grasp_props),
         )
         world.add_robot(robby, loc="bedroom")
-        robby.set_path_planner(
-            RRTPlanner(world, bidirectional=True, rrt_connect=False, rrt_star=True)
-        )
+        planner_config = {
+            "grid": None,
+            "world": world,
+            "bidirectional": False,
+            "rrt_connect": False,
+            "rrt_star": False,
+        }
+        rrt = PathPlanner("rrt", planner_config)
```

Comment:
As discussed offline, I like the `**kwargs` idea where the lowest-level planner actually has named arguments. It would change this to:

```python
rrt = PathPlanner("rrt", **planner_config)
```